from0k2bp.tex

```diff
@@ -224,22 +224,23 @@
 homomorphic;
 Pedersen}{Commitments; homomorphic; Pedersen}}\label{commitments-homomorphic-pedersen}}
 
-(I'm going to blatantly plagiarise myself for the much of this section,
```

Comment:
Wow you really don't like parentheses  :)

Reply:
Haha